config/core/lab-configs.yaml

```diff
@@ -31,6 +31,23 @@ labs:
             - ltp
             - preempt-rt
             - smc
+      - combination:
```

Comment:
One more thing to note, the jobs on staging.kernelci.org use the `kernelci` kernel tree.  So if you want lab-cip to keep running all the staging jobs, you'll need to add those branches too: `staging-mainline`, `staging-next`, `staging-stable`, `staging-cip` with the `kernelci` tree.  That's why other labs with a filter on trees have `kernelci` in their passlist.

Reply:
An exception to that, every day there's a regular mainline job run on staging but with a small subset of builds:
https://staging.kernelci.org/job/mainline/branch/master/

There's also a full linux-next job run once a week:
https://staging.kernelci.org/job/next/branch/master/

These jobs should match the current filters without adding the `kernelci` tree.

Reply:
Yea I was wondering about that :P

I guess the above will be enough for now. We can always re-expand if needed. Thanks for the info.

Reply:
Maybe at least add `kernelci` / `staging-cip`?

Reply:
Sure. Done.